src/cheri/app-cap-description.adoc

```diff
@@ -613,7 +613,7 @@ expanded base is 0 and top is 2^MXLEN^.
 [#null-cap-encoding,reftext="NULL",options=header,align=center,width="70%",cols="25,23,52"]
 |==============================================================================
 | Field    | Value  | Comment
-| Valid Tag                | zero   | Capability is not valid
+| Capability Tag           | zero   | Capability is not valid
```

Comment:
```suggestion
| {ctag}           | zero   | Capability is not valid
```

Reply:
I didn't change this one to use the macro due to capitalization. Could add a {CTAG} macro?

Reply:
possibly or just let it be lower case - note that varaible names aren't (normally) case sensiive and the handling is a bit buggy so I'd advise using a macro with a distinct name